docs/sources/whatsnew/whats-new-in-v9-3.md

```diff
@@ -110,6 +110,35 @@ $ grafana-cli user-manager conflicts validate-file <filepath>
 $ grafana-cli user-manager conflicts ingest-file <filepath>
 ```
 
+## LDAP - Role mapping improvements
+
+If you use an LDAP directory to authenticate to Grafana but prefer to assign organizations and roles in the Grafana UI
+or via API, you can now enable a configuration option to skip user organization roles synchronization with your LDAP
+directory.
+
+Use the [`skip_org_role_sync` LDAP authentication configuration option]({{< relref
+"../setup-grafana/configure-security/configure-authentication/ldap/#disable-org-role-synchronization" >}})
+when configuring LDAP authentication to prevent the synchronization between your LDAP groups and organization roles
+and make user roles editable from within Grafana.
+
+## Azure AD OAuth2 - New option to always fetch groups from the Graph API
+
+If you use Azure AD OAuth2 authentication and use `SecurityEnabled` groups that you don't want Azure to embed in the
+authentication token, you can force Grafana to use Microsoft's Graph API instead.
+
+Use the [`force_use_graph_api` configuration option]({{< relref
+"../setup-grafana/configure-security/configure-authentication/azuread/#force-fetching-groups-from-microsoft-graph-api" >}})
+when configuring Azure AD authentication to force Grafana to fetch
```

Comment:
Looks like the sentence is missing an ending.
```suggestion
when configuring Azure AD authentication to force Grafana to fetch groups using Graph API.
```

Reply:
Seems like something disappeared when I shuffled things around :sweat_smile: